backend/migrations/Version20250917182707.php

```
@@ -0,0 +1,59 @@
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20250917182707 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)